pkg/plugins/envvars/envvars.go

```diff
@@ -17,25 +17,41 @@ import (
 	"github.com/grafana/grafana/pkg/plugins"
 	"github.com/grafana/grafana/pkg/plugins/auth"
 	"github.com/grafana/grafana/pkg/plugins/config"
+	"github.com/grafana/grafana/pkg/services/featuremgmt"
 )
 
 const (
 	customConfigPrefix = "GF_PLUGIN"
 )
 
+// allowedHostEnvVarNames is the list of environment variables that can be passed from Grafana's process to the
+// plugin's process
+var allowedHostEnvVarNames = []string{
+	// Env vars used by net/http (Go stdlib) for http/https proxy
+	// https://github.com/golang/net/blob/fbaf41277f28102c36926d1368dafbe2b54b4c1d/http/httpproxy/proxy.go#L91-L93
+	"HTTP_PROXY",
+	"http_proxy",
+	"HTTPS_PROXY",
+	"https_proxy",
+	"NO_PROXY",
+	"no_proxy",
+}
+
 type Provider interface {
 	Get(ctx context.Context, p *plugins.Plugin) []string
 }
 
 type Service struct {
-	cfg     *config.Cfg
-	license plugins.Licensing
+	cfg      *config.Cfg
+	license  plugins.Licensing
+	features featuremgmt.FeatureToggles
 }
 
-func NewProvider(cfg *config.Cfg, license plugins.Licensing) *Service {
+func NewProvider(cfg *config.Cfg, license plugins.Licensing, features featuremgmt.FeatureToggles) *Service {
```

Comment:
Just wanted to highlight that in the areas where `*config.Cfg` is already available, you should be able to reach for features through that.

Reply:
that's now deprecated: https://github.com/grafana/grafana/blob/main/pkg/setting/setting.go#L364

Reply:
I'm referring to [our plugin config](https://github.com/grafana/grafana/blob/main/pkg/plugins/config/config.go#L46)!

Reply:
Ah, right :sweat_smile:  I will change that

Reply:
Done!